src/metabase/pulse/render/body.clj

```diff
@@ -34,12 +34,12 @@
 
 (def rows-limit
   "Maximum number of rows to render in a Pulse image."
-  20)
-
-(def cols-limit
-  "Maximum number of columns to render in a Pulse image."
   10)
 
+(def ^:dynamic *cols-limit*
+  "Maximum number of columns to render in a Pulse image. Defaults to infinity, so that columns are not truncated."
+  ##Inf)
```

Comment:
Using `##Inf` here is a bit of a hack to remove the column limit without having to refactor a bunch of other code to get rid of the column-limiting logic entirely. But we should probably do that cleanup eventually.